src/components/RouteSchedule/RouteSchedule.js

```diff
@@ -196,10 +196,9 @@ const defaultRenderFooter = (props) => {
   return <div className="rt-route-footer">{renderCopyright({ ...props })}</div>;
 };
 
-const defaultRenderLink = (title, text, url) => {
+const defaultRenderLink = (text, url) => {
   return (
     <div className="rt-route-copyright-link">
-      {title}
       {url ? (
         <a href={url} target="_blank" rel="noreferrer">
```

Comment:
We could put the 'title' parameter you remove in the title attribute of the link what do you think ?

Reply:
arf forget I just saw the result

Reply:
Indeed removing titles is a requiremnt now